src/screens/transaction-details/ThinSymbolRow.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Box, Columns, Inline, Text } from '@/design-system';
+
+type Props = {
+  icon: string;
+  title: string;
+  value: string;
+};
+
+export const ThinSymbolRow: React.FC<Props> = ({ icon, title, value }) => {
```

Comment:
not big on this component name, something like TransactionDetailsRow makes more sense to me but your call

Reply:
Wel, there will be a different kind of row, hence the Thin, but good to know I will figure out a different name for that